canonical-websites/snapcraft.io
# Various updates

## Done

- Add “Forum” as the last item in snapcraft.io’s top-level navigation
- Change the contents of the “Learn more” footer section to
  - [Forum](https://forum.snapcraft.io/)
  - [Real-time chat](https://rocket.ubuntu.com/channel/snapcraft)
  - [Snapcraft on Stack Overflow](http://stackoverflow.com/questions/ask?tags=snapcraft)
- Remove “Community” from the top-level navigation
- Delete the /community page, redirecting the URL to the front page

## QA

- Follow the instructions in README.md to get started
- Go to http://127.0.0.1:4000/ and make sure all the changes in the issue have been actioned
- Go to http://127.0.0.1:4000/community and make sure you’re redirected to the home page

## Issue / Card

Fixes #322
